### DISCUSSION TITLE
Update discussions / issues in the background and show an updated info

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#18444] [Mobile]
Show a little updated info window / notification if something on that page (e.g. issue or discussion) has updated (new post, closed etc) with a button to update the page.

At the moment you don't get notified if something changes. You have to scroll all the way up and pull it down to update a page

**Comment by eliperkins** [comment]
Hey @Zerotask, thanks for this feedback!

Seeing live updates when there are new comments, replies or answer to issues, pull requests, and discussions sounds like an awesome feature.

I've shared your notes about what you'd like to see on our internal tracking issue for live updates. Stay tuned!
